addons/hr/views/hr_views.xml

```diff
@@ -139,6 +139,10 @@
                                         <field name="company_id" groups="base.group_multi_company"/>
                                         <field name="user_id" string="Related User"/>
                                     </group>
+                                    <group string="Login Information" name="identification_group">
+                                        <field name="barcode"/>
```

Comment:
<button type="object" name="_generate_random_barcode" attrs="{'invisible':[('barcode','!=','')]}"/>

    def _default_random_barcode(self):
	for i in self: i.barcode = "".join(choice(digits) for i in range(8))